API.md

```diff
@@ -0,0 +1,525 @@
+    | `degree`            | `boolean` | `false` | Converts trigonometric values from radians to degrees.                                                   |
+    | `operators`         | `object`  | `{}`    | The list of operators to enable.                                                                         |
+
+-   `calculator.setDegreeMode(degree = true)`: Sets the engine to process the angles in degree (`true`) or in radian (`false`).
+-   `calculator.isDegreeMode()`: Tells if the engine process the angles in degree (`true`) or in radian (`false`).
+-   `calculator.setInstantMode(mode = true)`: Sets the engine to compute the expression instantaneously (`true`) or not (`false`).
+-   `calculator.isInstantMode()`: Tells if the engine must compute the expression instantaneously (`true`) or not (`false`).
+-   `calculator.setCorrectorMode(mode = true)`: Sets the engine to correct the expression before evaluating it (`true`) or not (`false`).
+-   `calculator.isCorrectorMode()`: Tells if the engine must correct the expression before evaluating it (`true`) or not (`false`).
+-   `calculator.getMathsEvaluator()`: Gets access to the mathsEvaluator.
+-   `calculator.getTokenizer()`: Gets access to the tokenizer.
+-   `calculator.reset()`: Resets the calculator.
+
+## <a name='Events'></a>Events
+
+The engine emits events each time a modification or an action apply to the calculator.
+
+| event            | parameters               | description                                                     |
+| ---------------- | ------------------------ | --------------------------------------------------------------- |
+| `configure`      | `config`                 | Notifies the maths evaluator has been configured.               |
+| `expression`     | `expression`             | Notifies the expression has changed.                            |
+| `position`       | `position`               | Notifies the position inside the expression has changed.        |
+| `variableadd`    | `name`, `value`          | Notifies a variable has been added.                             |
+| `variabledelete` | `name`                   | Notifies a variable has been removed.                           |
+| `variableclear`  |                          | Notifies all variables have been removed.                       |
+| `commandadd`     | `name`                   | Notifies a command has been registered.                         |
+| `commanddelete`  | `name`                   | Notifies a command has been removed.                            |
+| `commandclear`   |                          | Notifies all commands have been removed.                        |
+| `command`        | `name`, ...              | Notifies a command has been invoked.                            |
+| `command-<name>` | ...                      | Notifies the command `<name>` has been invoked.                 |
```

Comment:
Please check the usage here https://github.com/oat-sa/live-design-system/blob/9172a8d24d3d9151d2541df121af787b1784bee7/packages/components/calculator/Calculator.svelte#L439

There is space-separation event binding used

Reply:
Ok, sorry - it's binding to 2 events in one `on` call